test/.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "node": true
   },
   "parserOptions": {
-    "ecmaVersion": 6
+    "ecmaVersion": 6,
+     "sourceType": "module"
```

Comment:
this looks like weird indentation (could we get prettier to run over the config `.json`/`.js` files at top level?)

Reply:
I moved the prettyfier stuff to the Makefile and added .json files.

Reply:
Nice!